lldb/source/Expression/Materializer.cpp

```diff
@@ -662,12 +657,6 @@ class EntityVariableBase : public Materializer::Entity {
 
       CompilerType valobj_type = valobj_sp->GetCompilerType();
 
-#ifdef LLDB_ENABLE_SWIFT
-      if (SwiftASTContext::IsGenericType(valobj_type)) {
-        valobj_sp = valobj_sp->GetDynamicValue(lldb::eDynamicDontRunTarget);
-      }
-#endif // LLDB_ENABLE_SWIFT
```

Comment:
why is this no longer needed?

Reply:
It was very old code that had no longer any effect on the test suite.